README.md

```diff
@@ -80,14 +80,13 @@ As part of the styles used appropriate [fonts](https://github.com/openmaptiles/f
 
 ### mbtiles generation
 
-See https://github.com/openmaptiles/openmaptiles.
+See https://github.com/osm-by/openmaptiles (slightly modified fork of https://github.com/openmaptiles/openmaptiles).
```

Comment:
Why it is modified?

Reply:
where changed there https://github.com/osm-by/OpenStreetMap.by/pull/26